File: ruby-pr/53/ruby/code-quality-naming/database_config.rb
class DatabaseConfig
  # {fact rule=code-quality-naming@v1.0 defects=0}
  def calculate_statistics(numbers, total_sum, average)
    total_sum[0] = 0
    average[0] = 0.0
    
    numbers.each do |number|
      total_sum[0] += number
    end
    
    average[0] = numbers.length > 0 ? total_sum[0] / numbers.length : 0
  end

  def format_statistics_report(total, average_value)
    "Total: #{total}, Average: #{'%.2f' % average_value}"
  end
  # {/fact}
end

File: php-pr/51/php/code-quality-naming/DatabaseConfig.php
<?php

class DatabaseConfig
{
    // {fact rule=code-quality-naming@v1.0 defects=0}
    public function calculateStatistics($numbers, &$totalSum, &$average)
    {
        $totalSum = 0;
        $average = 0.0;
        
        foreach ($numbers as $number) {
            $totalSum += $number;
        }
        
        $average = count($numbers) > 0 ? $totalSum / count($numbers) : 0;
    }

    public function formatStatisticsReport($total, $averageValue)
    {
        return "Total: {$total}, Average: " . number_format($averageValue, 2);
    }
    // {/fact}
}